youtube_dl/extractor/televizeseznam.py

```diff
@@ -0,0 +1,85 @@
+# coding: utf-8
+from __future__ import unicode_literals
+
+import json
+
+from .common import InfoExtractor
+from ..compat import compat_urlparse
```

Comment:
`from ..utils import urljoin` based on what you use from urlparse

Reply:
Thanks